Zend/zend_portability.h

```diff
@@ -394,7 +394,7 @@ char *alloca();
 # define XtOffsetOf(s_type, field) offsetof(s_type, field)
 #endif
 
-#ifdef HAVE_SIGSETJMP
+#ifndef ZEND_WIN32
```

Comment:
If we keep the autoconf check, we may keep the `#ifdef HAVE_SIGSETJMP` here too?

Reply:
that s the thing, with this PR this constant is no longer defined as it is implicitly required on unix.

Reply:
is the autoconf check still useful?

Reply:
I would say yes, so it s detected at configure time rather than build's e.g. if a linux/dietlibc user.

Reply:
Oh right, I missed that you replaced AC_DEFINE with AC_MSG_ERROR. Makes sense now